packages/web/src/common/store/cache/store.test.ts

```diff
@@ -25,16 +25,38 @@ const initialConfirmerState = {
 }
 
 const MOCK_TIMESTAMP = 1479427200000
+vitest.mock('@audius/common/store', async () => ({
+  ...((await vitest.importActual('@audius/common/store')) as object),
+  CACHE_PRUNE_MIN: 1
+}))
 
-// beforeAll(() => {
-//   config.CACHE_PRUNE_MIN = 1
-//   vitest.spyOn(Date, 'now').mockImplementation(() => MOCK_TIMESTAMP)
-// })
+vitest.mock('services/remote-config/remote-config-instance', async () => {
+  return {
+    remoteConfigInstance: {
+      waitForRemoteConfig: vitest.fn(),
+      getRemoteVar: vitest.fn().mockImplementation((arg) => {
+        if (arg === IntKeys.CACHE_ENTRY_TTL) {
+          return Infinity
+        }
+        return undefined
+      })
+    }
+  }
+})
 
-// TODO: PAY-2602
-describe.skip('add', () => {
+const defaultProviders: StaticProvider[] = [
+  [getContext('remoteConfigInstance'), remoteConfigInstance],
+  [getContext('getFeatureEnabled'), () => false]
+]
+
+beforeAll(() => {
```

Comment:
comment helpful for this?

Reply:
I didn't write this, but what comment do you think makes sense here?

Reply:
the cache used to prune itself with a TTL or something like that, maybe that's what this was for? but i think it's straightforward enough w/o a comment